### ISSUE TITLE
Calendar does not resize with editor pane

### ISSUE THREAD

**Issue by mbafford** [coddingtonbear/obsidian-itinerary#17]
Obsidian v.0.13.23
Calendar Version 1.5.10
New editor mode (not legacy)
Readable line length setting: OFF

When I resize the preview window, the calendar does not resize to fit the adjusted space. Toggling edit/view doesn't resize, either. Reloading Obsidian does display with the resized calendar, which looks good.

Parts of the calendar do resize, but the actual "calendar" portion doesn't (on either week or month view, didn't test others):

![image](https://user-images.githubusercontent.com/713813/155248332-d51d4d4b-d1e9-4939-9cba-27cac5f43f99.png)

The resize works fine if the resizing is of the Obsidian main window, just not when resizing individual panes within Obsidian.

The plugin is a cool idea - many thanks for implementing it!

**Comment by coddingtonbear** [comment]
Yep, we're doing some tricky things to convince the calendar into updating its size in certain situations; here's an example: https://github.com/coddingtonbear/obsidian-itinerary/blob/master/src/render.ts#L111.  It's not likely I'll look into this in the foreseeable future, but that particular line is probably enough of a clue to help you or somebody else throw together a pull request fixing this.

Cheers!
